OWNERS

```diff
@@ -1,18 +1,17 @@
 reviewers:
-  - codificat
```

Comment:
stray diff? this feels like it doesn't match the content of the rest of the PR

Reply:
You're right.  I can remove this diff.